ui/details-panel/AnnotationsTab.jsx

```diff
@@ -263,5 +264,7 @@ AnnotationsTab.defaultProps = {
   selectedJob: null,
 };
 
-treeherder.directive('annotationsTab', ['reactDirective', '$injector', (reactDirective, $injector) =>
-  reactDirective(AnnotationsTab, undefined, {}, { $injector })]);
+treeherder.component('annotationsTab', react2angular(
+  AnnotationsTab,
+  ['classificationTypes', 'classifications', 'bugs', 'selectedJob'],
```

Comment:
Edit:
Ignore the below, I guess we'd hit this bug otherwise:
https://github.com/coatue-oss/react2angular/issues/72

---

The react2angular docs say that if the second argument isn't specified, then it automatically uses the PropTypes to calculate the props:
https://github.com/coatue-oss/react2angular#2-expose-it-to-angular

Since we pass the third argument, we have to pass the second, but it looks like so long as it's a falsey value, the automatic PropTypes behaviour will still be used:
https://github.com/coatue-oss/react2angular/blob/d0b8c336b2a52e05839636b7651dad959be76e7a/index.tsx#L23-L25

I think it will be easier to maintain if we use this automatic behaviour rather than manually specifying. This may also require fixing some of the current PropType errors seen in the browser console and when running the tests (NB: the runtime PropType validation only happens in development mode).

Reply:
Yeah, I tried it the documented way with no dice.  :)  Hopefully they'll fix that bug soon.